javascript/ql/src/semmle/javascript/frameworks/Express.qll

```diff
@@ -147,6 +147,48 @@ module Express {
       this.getRequestMethod() = that.getRequestMethod()
     }
   }
+  
+  /**
+   * A call that sets up a Passport router that includes the request object.
+   */
+  class PassportRouteSetup extends HTTP::Servers::StandardRouteSetup, CallExpr {
```

Comment:
It may be worth making this class and the other one `private`. That way client code won't end up accidentally depending on them, which makes it easier to change the implementation later on if we want to.

Reply:
I made the two classes private.